vllm/model_executor/layers/pooler.py

```diff
@@ -27,7 +28,7 @@ class Pooler(nn.Module):
     3. Returns structured results as `PoolerOutput`.
 
     Attributes:
-        pooling_type: The type of pooling to use (LAST, ALL, CLS).
+        pooling_type: The type of pooling to use (LAST, ALL, CLS, MEAN).
```

Comment:
```suggestion
        pooling_type: The type of pooling to use.
```
To avoid having to keep updating, let's just remove this. The type annotation should be informative enough.

Reply:
This still hasn't been updated yet.

Reply:
Sorry! Done.